.infra/workers.ts

```diff
@@ -19,6 +19,14 @@ export const workers: Worker[] = [
     topic: 'user-updated',
     subscription: 'user-updated-api',
   },
+  {
+    topic: 'user-updated',
+    subscription: 'user-updated-mailing',
+  },
+  {
+    topic: 'user-updated',
+    subscription: 'user-deleted-mailing',
+  },
```

Comment:
I think these subs already exist in the gateway. Am I right?
If so, Pulumi will not be able to create them. We might need to do some manual work here. Let's think about it later